server/cmd/mmctl/commands/channel_users.go

```diff
@@ -94,27 +94,33 @@ func channelUsersRemoveCmdF(c client.Client, cmd *cobra.Command, args []string)
 		return errors.Errorf("unable to find channel %q", args[0])
 	}
 
+	var result *multierror.Error
 	if allUsers {
 		if err := removeAllUsersFromChannel(c, channel); err != nil {
 			return err
 		}
 	} else {
 		for i, user := range getUsersFromUserArgs(c, args[1:]) {
-			removeUserFromChannel(c, channel, user, args[i+1])
+			err := removeUserFromChannel(c, channel, user, args[i+1])
+			if err != nil {
+				result = multierror.Append(result, err)
+			}
 		}
 	}
 
-	return nil
+	return result.ErrorOrNil()
 }
 
-func removeUserFromChannel(c client.Client, channel *model.Channel, user *model.User, userArg string) {
+func removeUserFromChannel(c client.Client, channel *model.Channel, user *model.User, userArg string) error {
 	if user == nil {
 		printer.PrintError("Can't find user '" + userArg + "'")
-		return
+		return fmt.Errorf("unable to find user %q", userArg)
 	}
 	if _, err := c.RemoveUserFromChannel(context.TODO(), channel.Id, user.Id); err != nil {
 		printer.PrintError("Unable to remove '" + userArg + "' from " + channel.Name + ". Error: " + err.Error())
+		return fmt.Errorf("unable to remove %q from %q. Error: %w", userArg, channel.Name, err)
```

Comment:
Nit pick: To avoid the duplication, you can also use the pattern with the tmp variable from https://github.com/mattermost/mattermost/blob/19c116511775166533d5ec6b7ec1f6e8e5336cc8/server/cmd/mmctl/commands/channel.go#L330-L333.

Reply:
In the entire file `PrintError` begins with a capital letter and have single quotes while `Errorf` begins with a lowercase letter and have double quotes. I thought this was intentional, as it was written by the same person at the same time.

Reply:
2/5 that the casing doesn't matter much. If we care, we can still convert to upper case in `PrintError`. Let's move the printing out of `removeUserFromChannel` to the caller `channelUsersRemoveCmdF`.

Reply:
Done. I added the same functionality to `addUserToChannel`